app/cdap/components/Connections/Browser/GenericBrowser/index.tsx

```diff
@@ -141,15 +145,23 @@ export function GenericBrowser({ selectedConnection }) {
           loading={loading}
         />
       </If>
-      <If condition={isEmpty && !error}>
-        <MuiAlert severity="info" className={classes.alert}>
-          No entities available
-        </MuiAlert>
+      <If condition={isEmpty && !loading}>
```

Comment:
Should this be visible in the event of an error which is shown in the ErrorBanner component below, on line 164?

Reply:
Yes, this matches the current behavior.